spotify_player/src/client/request.rs

```diff
@@ -38,6 +38,7 @@ pub enum ClientRequest {
     },
     Search(String),
     AddTrackToQueue(TrackId<'static>),
+    AddEpisodeToQueue(EpisodeId<'static>),
```

Comment:
Would love to combine these into a `AddItemToQueue(PlayableId<'static>)`, but can't do that yet as `PlayableId` doesn't `impl Debug` or `Clone`